### PR TITLE
Add correction to agent.filesystem metrics table

### PR DESCRIPTION
**Pull request by narcher7**
**Quality audit reminder**

Doc team: Use the quality checklist at http://goo.gl/forms/GaRzOmhzr5 to ensure that the 
content of the PR meets minimal writing and style requirements, and has received an 
appropriate subject matter expert (SME) review. Complete the checklist for significant
doc changes or additions that you submit or that are submitted by a non-doc staff contributor.

- Fixes issue #11

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-7488136e42/docs/rackspace-monitoring/v1/).

File: api-docs/tech-ref-info/check-types/agent-check-type-ref.rst [hunk 1]
```diff
@@ -188,8 +188,8 @@ Metrics
 +-----------------+--------------------------------------------------+----------+
 | Metric          | Description                                      | Type     |
 +=================+==================================================+==========+
-| avail           | Available space on the filesystem in kilobytes,  | Int64    |
-|                 | including reserved space.                        |          |
```
Comment (wolfdancer) [inline api-docs/tech-ref-info/check-types/agent-check-type-ref.rst:192]: "for normal users" seems to be a bit too vague, right?  how about

"including reserved space, which is how normal users evaluate the available space"

Comment (narcher7) [reply api-docs/tech-ref-info/check-types/agent-check-type-ref.rst:192]: "Reserved space" would mean the same thing as "used space" correct? Would I need to make this change on the line labeled "free" as well?

Comment (wolfdancer) [reply api-docs/tech-ref-info/check-types/agent-check-type-ref.rst:192]: Hi @DonNateR : The reserved space is the space reserved by the system.  Used space is the space used, not including the reserved space.  

Looking at the documentation, I think it assumes that the reader is familiar with these metrics.  I can talk to our engineers to see how we can clarify the documentation further.

Comment (narcher7) [reply api-docs/tech-ref-info/check-types/agent-check-type-ref.rst:192]: Hi, @wolfdancer: That's what I assumed. Might be a non-issue. Let me know if you hear back from the engineers.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-78f2f4b252/docs/rackspace-monitoring/v1/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-69690be587/docs/rackspace-monitoring/v1/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-5ccd5316a4/docs/rackspace-monitoring/v1/)

**Comment by cyrichardson** [comment]
@DonNateR Let me know if you are ready for a review/merge on this one. Thanks!

**Comment by wolfdancer** [comment]
@cyrichardson Please wait until I hear back from engineering.

**Comment by cyrichardson** [comment]
@wolfdancer Will do!

**Comment by narcher7** [comment]
@cyrichardson: I'll make a separate PR to address the table formatting while we wait for Shane.

**Comment by cyrichardson** [comment]
@DonNateR OK!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-5ccd5316a4/docs/rackspace-monitoring/v1/). :bow:

**Comment by wolfdancer** [comment]
+1

@DonNateR @cyrichardson : Based on the information I got from the engineer, this PR is good to be merged.  I'll submit another PR to add more explanation of what these metrics and some related concepts are.  Thanks for your help and waiting.

**Comment by cyrichardson** [comment]
Thanks, @wolfdancer! 
@DonNateR Looks like some conflicts to resolve before merging.

**Comment by narcher7** [comment]
@cyrichardson: Conflict is with the last PR we merged. Probably easier to close this one. I'll open a new one for you to review.

**Comment by narcher7** [comment]
@cyrichardson: Since the main changes I made to this PR regarded the table formatting (which was fixed on PR #16), I'll wait for Shane to make his PR before I change any content on the table itself.

**Comment by cyrichardson** [comment]
@DonNateR  OK!

**Pull request closed without merging**